src/dss/logs.py

```diff
@@ -0,0 +1,93 @@
+from lightkube import Client
+from lightkube.core.exceptions import ApiError
+from lightkube.resources.apps_v1 import Deployment
+from lightkube.resources.core_v1 import Pod
+
+from dss.config import DSS_NAMESPACE, MLFLOW_DEPLOYMENT_NAME
+from dss.logger import setup_logger
+
+# Set up logger
+logger = setup_logger("logs/dss.log")
+
+
+def get_logs(parts: str, name: str, lightkube_client: Client) -> None:
```

Comment:
What about keeping the argument as `notebook_name`, since that's what the caller function uses and it makes for a more transparent name.

Reply:
I don't agree ... the argument is None if --mlflow or --all is provided.

Reply:
yes, but in those cases it's not used at all. The argument always represents a notebook's name.

Reply:
It ius actually used in the error message output when there cam be also --mlflow nad --all not just for notebooks :)